### DISCUSSION TITLE
ScriptReference/EditorGUIUtility-isProSkin

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#673] [Notes]
# ScriptReference/EditorGUIUtility-isProSkin

https://docs.unity3d.com/ScriptReference/EditorGUIUtility-isProSkin.html

**Comment by andreasIsCoding** [comment]
There is an undocumented method that you can use to switch skins `InternalEditorUtility.SwitchSkinAndRepaintAllViews ();`
Despite being in the internal name space, you do not need reflection to call this method.
